components/Image/FullWidthImage.tsx

```diff
@@ -39,6 +41,7 @@ export const FullWidthImage = ({
           alt={alt || ''}
           width={originalWidth}
           height={originalHeight}
+          fetchPriority={fetchPriority}
```

Comment:
Add `fetchPriority` prop to FullWidthImage and AspectRatioImage so I can add fetchPriority="high" to hero/header images. I have to add it manually since apparently it still isn't included in `React.HTMLAttributes<HTMLImageElement>`